platforms/web/package.json

```diff
@@ -56,6 +56,7 @@
     "allure-commandline": "^2.17.2",
     "babel-plugin-transform-typescript-metadata": "^0.3.2",
     "codeceptjs": "3.5.5",
+    "core-js": "^3.37.0",
```

Comment:
Do we maybe want to make this more specific, since we only use 2 js files?

Reply:
I get your point. But unfortunately that is not possible. I did a Google search and also used ChatGTP to verify.